jax/experimental/jax2tf/tests/jax2tf_test.py

```diff
@@ -1741,9 +1730,10 @@ def f_jax(x):
       native_serialization=True,
       native_serialization_platforms=("cpu", "cuda", "tpu"))
     for tf_device in self.__class__.tf_devices:
+      logging.info(
```

Comment:
As in the other test file, this should probably be a subtest.

Reply:
Done